sdcm/nemesis.py

```diff
@@ -746,6 +748,25 @@ def disrupt_validate_hh_short_downtime(self):
         assert self.tester.hints_sending_in_progress() is False, "Hints are sent too slow"
         self.tester.verify_no_drops_and_errors(starting_from=start_time)
 
+    def disrupt_snapshot_operations(self):
+        self._set_current_disruption('SnapshotOperations')
+        result = self._run_nodetool('nodetool snapshot', self.target_node)
+        self.log.info(result)
+        if result.stderr:
+            self.tester.fail(result.stderr)
+        snapshot_name = re.findall('(\d+)', result.stdout, re.MULTILINE)[0]
+        result = self._run_nodetool('nodetool listsnapshots', self.target_node)
+        if snapshot_name in result.stdout and not result.stderr:
+            self.log.info('Snapshot %s created' % snapshot_name)
+            self.log.info(result)
```

Comment:
if you are logging results in both cases, move it away from `if...else...` structure

Reply:
additionally `result` is kind of `debug` level message. So use `self.log.debug(...)`.